src/tests/operate.test.js

```diff
@@ -0,0 +1,5 @@
+import operate from '../logic/operate';
+
+test('Test operate Logic (Addition) | operate(6, 9, +)', () => {
+  expect(operate(6, 9, '+')).toEqual('15');
+});
```

Comment:
- [ ] I suggest that you increase tests here you tested the `+` operator only please test all operators you think a calculator can perform